ddf-rpc-api/src/main/java/com/connexta/ddf/catalog/direct/CatalogMethods.java

```diff
@@ -625,17 +599,144 @@ private Object create(Map<String, Object> params) {
 
   private ImmutablePair<Attribute, String> getAttribute(String name, Object value) {
 
-    AttributeDescriptor ad =
-        attributeRegistry
-            .lookup(name)
-            .orElseGet(
-                () ->
-                    new AttributeDescriptorImpl(
-                        name, true, true, false, true, BasicTypes.STRING_TYPE));
-
+    AttributeDescriptor ad = attributeRegistry.lookup(name).orElse(null);
+    // TODO (RCZ) - I'm proposing removing this orElseGet to create a new string attribute
+    //  descriptor
+    //  in the case that one cant be found. This silent behavior will make your thing stored, but
+    //  may result in confusion on why its a string. i would propose that we do what the catalog
+    //  framework does by default which is not storing that attribute.
+    //            .orElseGet(
+    //                () ->
+    //                    new AttributeDescriptorImpl(
+    //                        name, true, true, false, false, BasicTypes.STRING_TYPE));
+
+    if (ad == null) {
```

Comment:
Keeping separate now that i've added specific logging to the Attribute Descriptor being missing (which should not log in case of a null attribute value)